application/src/main/java/dev/stratospheric/todoapp/config/AsyncConfig.java

```diff
@@ -0,0 +1,22 @@
+package dev.stratospheric.todoapp.config;
+
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.context.annotation.Primary;
+import org.springframework.core.task.TaskExecutor;
+import org.springframework.scheduling.annotation.EnableAsync;
+import org.springframework.scheduling.concurrent.ThreadPoolTaskExecutor;
+
+@EnableAsync
+@Configuration
+public class AsyncConfig {
+
+  @Bean
+  @Primary
+  public TaskExecutor taskExecutor() {
```

Comment:
With Spring Cloud AWS, we have various `TaskExecutor`. Without making one primary, Spring Boot logs an info message and picks potentially one of the executor of Spring Cloud AWS